agent/main.go

```diff
@@ -155,7 +170,8 @@ func main() {
 		k8sInstaller = nil
 		logger.Info("skip-installation flag set, skipping installer initialisation")
 	} else {
-		k8sInstaller, err = installer.New(downloadpath, logger)
+		// increasing installer log level to 1, so that it wont be logged by default
+		k8sInstaller, err = installer.New(downloadpath, logger.V(1))
```

Comment:
This hardcoding will not allow us to see any logs even if a higher log verbosity level is provided.

Reply:
`logger.V(1)` specifies the minimum verbosity level required to view the logs, so installer logs will be printed for any `v ` value `>= 1`